cve_bin_tool/cvedb.py

```diff
@@ -456,8 +454,13 @@ def populate_db(self) -> None:
         we'll need a better parser to match those together.
         """
 
-        self.store_epss_data()
         self.populate_metrics()
+        """
+        EPSS uses metrics table to get the EPSS metric id.
+        It can't be ran before creation of metrics table.
+        """
```

Comment:
This one actually doesn't need to be a string, it can be a comment.  Docstrings are a special case of a string appearing immediately after a `def` function definition (at least for functions) but anything else inside the function can probably be a comment.

```suggestion
        # EPSS uses metrics table to get the EPSS metric id.
        # It can't be ran before creation of metrics table.
```